### PR TITLE
Changes to the move rotate and scale

### PR DESCRIPTION
**Pull request by aaguilea**
Also removed some of the IDs to the Crc32
Signed-off-by: aaguilea <aaguilea@amazon.com>

### CODE REVIEW

**Review by jonawals — APPROVED** [review]

File: Code/Editor/MainWindow.cpp [hunk 1]
```diff
@@ -731,32 +732,84 @@ void MainWindow::InitActions()
...
+            });
+    am->AddAction(AzToolsFramework::EditModeScale, tr("Scale"))
         .SetIcon(Style::icon("Scale"))
         .SetApplyHoverEffect()
         .SetShortcut(tr("3"))
         .SetToolTip(tr("Scale (3)"))
         .SetCheckable(true)
         .SetStatusTip(tr("Select and scale selected object(s)"))
-        .RegisterUpdateCallback(cryEdit, &CCryEditApp::OnUpdateEditmodeScale);
+        .RegisterUpdateCallback([](QAction* action)
```
Comment (hultonha) [inline Code/Editor/MainWindow.cpp:794]: As part of this chance can you also delete the member functions on `CCryEditApp` that are no longer used (e.g. `&CCryEditApp::OnUpdateEditmodeScale` etc..

File: Code/Editor/ViewportTitleDlg.cpp [hunk 2]
```diff
@@ -953,13 +953,13 @@ void CViewportTitleDlg::CheckForCameraSpeedUpdate()
...
-    MainWindow::instance()->GetActionManager()->GetAction(ID_SNAP_TO_GRID)->trigger();
+    MainWindow::instance()->GetActionManager()->GetAction(AzToolsFramework::SnapToGrid)->trigger();
 }
 
 void CViewportTitleDlg::OnAngleSnappingToggled()
 {
     m_angleSizeActionWidget->setEnabled(m_enableAngleSnappingAction->isChecked());
-    MainWindow::instance()->GetActionManager()->GetAction(ID_SNAPANGLE)->trigger();
+    MainWindow::instance()->GetActionManager()->GetAction(AzToolsFramework::SnapAngle)->trigger();
 }
```
Comment (hultonha) [inline Code/Editor/ViewportTitleDlg.cpp:963]: As a follow-up change after this PR I wonder if we can get rid of these functions too, they might not actually be needed anymore... I can't quite recall how/when they're called but we might be able to simplify the logic here...

**Review by hultonha — COMMENTED** [review]
Looks good! I just think there's some code that could be deleted we're missing, but otherwise nice change! 🙂

File: Code/Editor/MainWindow.cpp (hunk 1 above)
Comment (aaguilea) [reply Code/Editor/MainWindow.cpp:794]: Done

File: Code/Editor/ViewportTitleDlg.cpp (hunk 2 above)
Comment (hultonha) [reply Code/Editor/ViewportTitleDlg.cpp:963]: Yeah totally, it just reminded me and I thought I'd mention it 🙂

**Review by hultonha — APPROVED** [review]
Nice! 😄 Great little clean-up! Thanks for making the updates 👍

**Pull request merged**